app/views/common/403.html.erb

```diff
@@ -2,7 +2,14 @@
   <h2><%= _("Permission denied") %></h2>
 
   <p><strong><%= _("You are not authorized to perform this action") %></strong></p>
-  <p><%= _("Please request the required privileges from a foreman administrator") %></p>
+  <p><%= _("Please request one of the required permissions from a Foreman administrator listed below:") %>
```

Comment:
Sorry, I should've just given you the correct sentence on the first time.  "Please request one of the required permissions listed below from a Foreman administrator:"